stablehorde_api/models.py

```diff
@@ -1,22 +1,79 @@
 from typing import Optional, Sequence
+import logging
 
 import msgspec
 
+class ModelPayloadLorasStable(msgspec.Struct):
+    name: str # The exact name or CivitAI ID of the LoRa.
+    model: int | None = None # The strength of the LoRa to apply to the SD model.
+    clip: int | None = None # The strength of the LoRa to apply to the clip model.
+
+    # If set, will try to discover a trigger for this LoRa which matches or
+    # is similar to this string and inject it into the prompt.
+    # If 'any' is specified it will be pick the first trigger.
+    inject_trigger: str | None = None
+
+    def to_dict(self):
+        return {f: getattr(self, f) for f in self.__struct_fields__ if getattr(self, f) is not None}
+
+
+class ModelPayloadTextualInversionsStable(msgspec.Struct):
+    name: str
+    inject_ti: str = "prompt" # prompt/negprompt
```

Comment:
Formatting - 2 spaces before inline comment
```suggestion
    inject_ti: str = "prompt"  # prompt/negprompt
```